src/fuzzmanager.rs

```diff
@@ -0,0 +1,108 @@
+use serde_json::map::Map;
+
+pub use crate::defs::*;
+
+impl FMStats {
+
+    pub fn new(coverage: &Vec<i64>) -> Self {
+        let mut covered = 0;
+        let mut total = 0;
+
+        for x in coverage.iter() {
+            let x = *x;
+            total += (x >= 0) as usize;
+            covered += (x > 0) as usize;
+        }
+        let missed = total - covered;
+
+        Self {
+            total,
+            covered,
+            missed,
+            percent: Self::get_percent(covered, total),
+        }
+    }
+
+    pub fn add(&mut self, other: &Self) {
```

Comment:
I don't like that this method leaves the object in an unfinished state (as the percentage is not updated), but I see you did that to avoid recalculating the percentage all the time for dirs.
Maybe add a comment to say that after calling this method, you need to call set_percent too.